### ISSUE TITLE
Performance Gap on Argoverse 2 Validation Set

### ISSUE THREAD

**Issue by Yipeng-Zhou** [HKUST-Aerial-Robotics/SIMPL#15]
Thanks for sharing this perfect work!

After evaluating the pre-trained model on Argoverse 2 validation set using the default configuration, I found a performance gap between the results I got and the data in the table IV (the last row). Here is the comparison:
![Screenshot from 2024-05-31 18-55-20](https://github.com/HKUST-Aerial-Robotics/SIMPL/assets/104423272/a4f10e0c-fcaf-4b28-a7e2-7a3764bc3f81)
This issue does not occur on the Argoverse 1 validation set.:
![Screenshot from 2024-05-31 18-58-39](https://github.com/HKUST-Aerial-Robotics/SIMPL/assets/104423272/9d192e01-12a4-4528-a293-61d35ee0b459)

Looking forward to your reply!

**Comment by MasterIzumi** [comment]
@Yipeng-Zhou Hi, thanks for your feedback! Actually, the released checkpoint for AV2 is not the one we used in the paper. I will check this issue and upload the correct one soon.

**Comment by MasterIzumi** [comment]
Sorry for the late reply. Unfortunately, I cannot find the exact checkpoint for the experiment. So I re-train the network and get a checkpoint that achieves similar performance. I have updated the weight at `saved_models/simpl_av2_bezier_ckpt-fix.tar`, and the quantitative result on AV2 validation set is:
```
Validation set finish, cost 72.22 secs
-- minade_1: 2.045 minfde_1: 5.594 mr_1: 0.663 brier_fde_1: 5.594 minade_k: 0.777 minfde_k: 1.452 mr_k: 0.196 brier_fde_k: 2.069
```
If you have any questions, please let me know.
Again, thanks for your valuable feedback!
